### PR TITLE
Fix example

### PR DESCRIPTION
**Pull request by b-luu**
## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [ ] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

File: website/docs/reference/project-configs/packages-install-path.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ By default, dbt will install packages in the `dbt_packages` directory, i.e. `pac
 <File name='dbt_project.yml'>
 
 ```yml
-module-path: packages
+modules-path: packages
```
Comment (runleonarun) [inline website/docs/reference/project-configs/packages-install-path.md]:
```suggestion
packages-install-path: dbt_packages
```

**Review by runleonarun — CHANGES_REQUESTED** [review]
@b-luu Thank you for submitting this change! Our v1.1 release changed this value so I made that suggestion in the code review. If that looks good to Jeremy, we can merge this. 

@jtcohen6 Just need a quick 👍🏻 if this looks correct. TY!

**Review by mirnawong1 — APPROVED** [review]

**Comment by b-luu** [comment]
Thanks @runleonarun & @mirnawong1 :pray: 

I guess we're waiting on @jtcohen6's :+1: 
Is that why you've blocked merging with the "requested changes" review @runleonarun ?

**Comment by nghi-ly** [comment]
many apologies, @b-luu ! with our recent reorganization of the docs, it caused a lot of PRs to have merge conflicts -- this unfortunately was one of them. we have been working our way through the list to resolve them.  

looks like PR #2435 has merged a fix for this same issue. i will go ahead and close this one. please don't hesitate to reopen if there's still something remaining. again, apologies for the delay!

**Pull request closed without merging**